src/main.js

```diff
@@ -1 +1,28 @@
-import './main.scss'
\ No newline at end of file
+require('dotenv').config()
+import './main.scss'
+
```

Comment:
About the commit massage, perhaps we don't need to add the word "unfinished" because we all already known the code it in progress 😄 . Thus just "Add function buildJobDropdown" is enough.

Reply:
รับทราบค่า